docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,373 @@
+  </TabItem>
+  <TabItem label="Cluster">
+    ```ini
+    tls-port 7001
+    port 0
+
+    cluster-enabled yes
+
+    tls-replication yes
+    tls-cluster yes
+
+    aclfile /path/to/users.acl
+
+    masterauth GENERATED_STRONG_PASSWORD
+    masteruser replica-user
+
+    tls-cert-file /usr/local/etc/redis/certs/server.crt
+    tls-key-file /usr/local/etc/redis/certs/server.key
+    tls-ca-cert-file /usr/local/etc/redis/certs/server.cas
+    tls-protocols "TLSv1.2 TLSv1.3"
+    ```
+  </TabItem>
+</Tabs>
+
+Once mutual TLS has been enabled, you will no longer be able to connect to
+the cluster without providing a valid client certificate. You can use the
+`tls-auth-clients optional` setting to allow connections
+from clients that do not present a certificate.
+
+See [TLS Support](https://redis.io/topics/encryption)
+in the Redis documentation for more details.
+
+<Admonition type="note" title="Create Redis Cluster">
+  When you're configuring Redis Cluster you need to create the cluster using `redis-cli`.
+
+  Use the following command to create the cluster. Please note `redis-cli --cluster create` accepts only IP addresses.
+
+  ```shell
+  export REDISCLI_AUTH=STRONG_GENERATED_PASSWORD
+  export CERTS_DIR=/path/to/certs/
+  export IP1=10.0.0.1 # update with the real node 1 IP
+  export IP2=10.0.0.2 # update with the real node 2 IP
+  export IP3=10.0.0.3 # update with the real node 3 IP
+  export IP4=10.0.0.4 # update with the real node 4 IP
+  export IP5=10.0.0.5 # update with the real node 5 IP
+  export IP6=10.0.0.6 # update with the real node 6 IP
+
+  redis-cli --user alice --cluster-replicas 1 --tls --cluster-yes \
+  --cluster create ${IP1}:7001 ${IP2}:7002 ${IP3}:7003 ${IP4}:7004 ${IP5}:7005 ${IP6}:7006 \
+  --cacert ${CERTS_DIR}/server.cas --key ${CERTS_DIR}/server.key --cert ${CERTS_DIR}/server.crt
+  ```
+
+</Admonition>
+
+<Admonition type="note" title="Redis Cluster Only">
+
+  To enable Redis cluster mode in Teleport, add the `mode=cluster` parameter to the connection URI in
+  your Teleport Database Service config file.
+
+  ```yaml
+  databases:
+    - name: "redis-cluster"
+      uri: "rediss://redis.example.com:6379?mode=cluster"
+  ```
+
+</Admonition>
+
+## Step 5/5. Connect
+
+Log into your Teleport cluster and see available databases:
+
+<Tabs>
+  <TabItem scope={["enterprise", "oss"]} label="Self Hosted">
+    ```code
+    $ tsh login --proxy=teleport.example.com --user=alice
+    $ tsh db ls
+    # Name          Description     Labels
+    # ------------- --------------- --------
+    # example-redis Example Redis   env=dev
+    ```
+  </TabItem>
+  <TabItem scope={["cloud"]} label="Cloud">
+    ```code
+    $ tsh login --proxy=mytenant.teleport.sh --user=alice
+    $ tsh db ls
+    # Name          Description     Labels
+    # ------------- --------------- --------
+    # example-redis Example Redis   env=dev
+    ```
+  </TabItem>
+</Tabs>
+
+To connect to a particular database instance, first retrieve its certificate
```

Comment:
From here until the "Supported Redis commands" header, nearly every line appears to be another box, which makes this part of the guide slightly hard to read. I think we can remove the `<Admonition></Admonition>` tags from the tip and the two notes while preserving the text as regular body text.

Reply:
I personally think that the current version is hard to follow and it's easy to miss a step as there are many "Only Redis Cluster" blocks. What do you think about splitting this guide into two: Redis and Redis Cluster as a separate pages?

Reply:
That sounds good to me! What do you think, @r0mant @smallinsky?